pkg/fe_controller/fe_pod.go

```diff
@@ -24,6 +24,13 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	meta_path = "/opt/starrocks/fe/meta"
```

Comment:
Is it a standard option to deploy under `/opt/`

Reply:
the path is typical for container deployment, it is not absolute we can use a different path.